streams/us.m3u

```diff
@@ -1760,3 +1760,255 @@ https://wtvjlive-f.akamaihd.net/i/MIAMI_STREAM1@19309/master.m3u8
+#EXTINF:-1 tvg-id="WFXTDT1.us" status="",FOX Boston MA (720p)
+http://51.161.118.146:8080/ISG03_FOX_BOSTON_MA_WFXT/video.m3u8
+#EXTINF:-1 tvg-id="WFLDDT1.us" status="",FOX Chicago IL (720p)
+http://51.161.118.146:8080/ISG03_FOX_CHICAGO_IL_WFLD/video.m3u8
+#EXTINF:-1 tvg-id="WJWDT1.us" status="",FOX Cleveland OH (540p)
+http://51.161.118.146:8080/ISG03_FOX_CLEVELAND_OH_WJW/video.m3u8
+#EXTINF:-1 tvg-id="KDFWDT1.us" status="",FOX Dallas TX (720p)
+http://51.161.118.146:8080/ISG03_FOX_DALLAS_TX_KDFW/video.m3u8
+#EXTINF:-1 tvg-id="KDVRDT1.us" status="",FOX Denver CO (720p)
+http://51.161.118.146:8080/ISG03_FOX_DENVER_CO_KDVR/video.m3u8
+#EXTINF:-1 tvg-id="FOXMountain.us" status="",FOX Detroit MI (720p)
+http://51.161.118.146:8080/ISG03_FOX_DETROIT_MI_WJBK/video.m3u8
+#EXTINF:-1 tvg-id="WTICDT1.us" status="",FOX Hartford CT (720p)
+http://51.161.118.146:8080/ISG03_FOX_HARTFORD_CT_WTIC/video.m3u8
+#EXTINF:-1 tvg-id="KRIVDT1.us" status="",FOX Houston TX (720p)
+http://51.161.118.146:8080/ISG03_FOX_HOUSTON_TX_KRIV/video.m3u8
+#EXTINF:-1 tvg-id="WXINDT1.us" status="",FOX Indianapolis IN (720p)
+http://51.161.118.146:8080/ISG03_FOX_INDIANAPOLIS_IN_WXIN/video.m3u8
+#EXTINF:-1 tvg-id="WFOXDT1.us" status="",FOX Jacksonville FL (720p)
+http://51.161.118.146:8080/ISG03_FOX_JACKSONVILLE_FL_WFOX/video.m3u8
+#EXTINF:-1 tvg-id="WSVNDT1.us" status="",FOX Miami FL (720p)
+http://51.161.118.146:8080/ISG03_FOX_MIAMI_FL_WSVN/video.m3u8
+#EXTINF:-1 tvg-id="WITIDT1.us" status="",FOX Milwaukee WI (720p)
+http://51.161.118.146:8080/ISG03_FOX_MILWAUKEE_WI_WITI/video.m3u8
+#EXTINF:-1 tvg-id="WVUEDT1.us" status="",FOX New Orleans LA (720p)
+http://51.161.118.146:8080/ISG03_FOX_NEW_ORLEANS_LA_WVUE/video.m3u8
+#EXTINF:-1 tvg-id="WOFLDT1.us" status="",FOX Orlando FL (720p)
+http://51.161.118.146:8080/ISG03_FOX_ORLANDO_FL_WOFL/video.m3u8
+#EXTINF:-1 tvg-id="KSAZDT1.us" status="",FOX Phoenix AZ (720p)
+http://51.161.118.146:8080/ISG03_FOX_PHOENIX_AZ_KSAZ/video.m3u8
+#EXTINF:-1 tvg-id="KTXLDT1.us" status="",FOX Sacramento CA (720p)
+http://51.161.118.146:8080/ISG03_FOX_SACRAMENTO_CA_KTXL/video.m3u8
+#EXTINF:-1 tvg-id="KSWBDT1.us" status="",FOX San Diego CA (720p)
+http://51.161.118.146:8080/ISG03_FOX_SAN_DIEGO_CA_KSWB/video.m3u8
+#EXTINF:-1 tvg-id="KCPQDT1.us" status="",FOX Seattle WA (720p)
+http://51.161.118.146:8080/ISG03_FOX_SEATTLE_WA_KCPQ/video.m3u8
+#EXTINF:-1 tvg-id="WTTGDT1.us" status="",FOX Washington DC (720p)
+http://51.161.118.146:8080/ISG03_FOX_WASHINGTON_DC_WTTG/video.m3u8
+#EXTINF:-1 tvg-id="FS1.us" status="",FS1 (720p)
+http://104.238.221.56:8080/live/823868279/9507295574/6014.m3u8?token=Q0dRVxcMEAMbCFkFVABTAVAAVAUBVlUMUAdSVgcDDFtaVl1UB1IDDgEbTRBDQBdXVF5vCFBACQ4CBFAbQERGXEo6UFJGXhAIDApRBggWSRBBWl0EFlgDFBBFDVNACxcLDFcKAkZIEFxBTQRAXlUJbVFWRghXBxECXEAIW04TWldnAVxbBQpWGwMbUBAcFgxBRREKD0EOXxQQVw1DEFBBXBpfGwVQXAIbFRsCXUVaEUBMEQpDYTERFBBQHEMHX0ZQVwsbDEYJAUwBG00QWUc6QFBARBNRA15dQBdeFVITGRtVBEFpBwtcV1xaFVtfWhYQDxEFQxhAXldcXBBYEG5FUFxHAxRdVwMOG0Q=
```

Comment:
FOX Sports Channels Are Not Allowed (Blacklisted Due to DMCA, See [Below](https://github.com/iptv-org/database/blob/master/data/blocklist.csv)